vault/dendron.tutorial.taking-notes.md

```diff
@@ -1,53 +1,91 @@
 ---
 id: 4RFQMNA91cRW12VJl_PFd
 title: Taking Notes
-desc: Taking Notes
-updated: 1636527192313
+desc: Creating notes, understanding hierarchy, and using Lookup to quickly find your notes
+updated: 1639602105376
 created: 1625563944736
 nav_order: 1
 ---
 
 ### Create a Note
 
-To create a note, use `Ctrl+L` / `Cmd+L` to bring up Dendron's lookup. This is a shortcut for the `Dendron: Lookup` command.
+To create a note, run `Dendron: Lookup` (`Ctrl+L` / `Cmd+L`) to bring up the [[Lookup Bar|dendron://dendron.dendron-site/dendron.topic.lookup]].
 
-> 💡 The lookup command is the main way to interact with Dendron. It is used for both looking up your notes and creating new notes. When you do a lookup on a note that hasn't been created, Dendron will create it for you. Remember the `Ctrl+L` / `Cmd+L` shortcut.
+![[dendron://dendron.dendron-site/asset.preview#lookup,1:#*]]
 
-> 🌱 Try it yourself - bring up the lookup bar with `Ctrl+L` / `Cmd+L`. Type `recipes` and hit Enter. Afterward, you can use the `Ctrl+Tab` / `Cmd+Tab` shortcut to get back to the previous note.
+> 💡 **TIP:** The lookup command is the main way to interact with Dendron. It's used for looking up and creating notes. When you do a lookup on a note that hasn't been created, Dendron will create it for you.
 
-This should have created a note named `recipes.md`. Notes in Dendron are just plaintext markdown. They live in your file system and are portable across any platform, which makes it very easy to import/ export your notes and even to publish your notes to a hosted website. You can try to add some content to the body of your `recipes` note now.
+> 🌱 **ACTION:** Try it yourself! Open the lookup bar with `Ctrl+L` / `Cmd+L`. Type `recipes` and hit `Enter`. Afterward, you can use the `Ctrl+Tab` / `Cmd+Tab` shortcut to get back to the previous note you were viewing.
```

Comment:
Its weird that the action is this this far below when the header of this section is "Create a note". I would restructure this section.

# Taking Notes

## Lookup

The Lookup Bar is the main way to interact with Dendron. You can use it for both finding notes and creating new notes.

![[lookup-gif]]

## Creating a Note

> **Action** Try it yourself...